packages/app/src/breadcrumbs/utils.ts

```diff
@@ -0,0 +1,28 @@
+import type { FeedbackContext } from './models';
+
+export function getFeedbackMailto(
```

Comment:
Not sure if this is worth documenting/exporting.

I would inline this in the `getFeedbackURL` of `demo` and be done with it

Reply:
Hmm but then you'll have to re-implement it in `jupyter-h5web` 😄

Reply:
Didn't think about this :clown_face: